Modules/Devices/test/Devices.Domain.Tests/TierNameTests.cs

```diff
@@ -30,6 +30,6 @@ public void Cannot_create_tier_name_with_invalid_name_length(int length)
 
         isTierNameInvalid.Should().BeTrue();
         errorCode.Should().Be("error.platform.validation.invalidTierName");
-        errorMessage.Should().Be($"Tier Name length must be between {TierName.MINIMUM_LENGTH} and {TierName.MAX_LENGTH}");
+        errorMessage.Should().Be($"Tier Name length must have a length between {TierName.MIN_LENGTH} and {TierName.MAX_LENGTH}");
```

Comment:
Damn. I'm too tired. Didn't see that the word "length" was already contained in the message. So you should revert the change. Sorry :)